frontend/package.json

```diff
@@ -9,29 +9,32 @@
     "lint:fix": "vue-cli-service lint "
   },
   "dependencies": {
-    "axios": "0.18.1",
-    "vue": "2.6.10",
-    "vue-router": "3.0.2",
-    "vuelidate": "0.7.4",
-    "vuetify": "1.5.14",
-    "vuex": "3.0.1"
+    "axios": "^0.18.1",
+    "sass": "^1.22.0",
+    "vue": "^2.6.10",
+    "vue-router": "^3.0.2",
+    "vuelidate": "^0.7.4",
+    "vuetify": "^2.0.0",
+    "vuex": "^3.0.1",
+    "webpack": "^4.39.3"
```

Comment:
how does it work before `webpack` in dependencies?

Reply:
We don't use vuetify-loader (webpack plugin) right now. I don't know if it wasn't required in older version of vuetify, or it works becaouse we have simple components structure. Nevertheless when i tried to add vuetify classes in dashboard subcomponent, vuetify couldn't load them correctly. In addition vuetify-loader is recomended solution for treeshaking in current vuetify docs https://vuetifyjs.com/en/customization/a-la-carte